trees/python/binary-tree.py

```diff
@@ -0,0 +1,66 @@
+from collections import deque
+
+class Node(object):
```

Comment:
Following [PEP8](https://www.python.org/dev/peps/pep-0008/), there should be a 2 line separation between your import statement and rest of your code.

Reply:
To automate this process try:
```
python3 -m pip install black isort  # this line only needs to be done once
black my_file.py
isort --profile black my_file.py
```

Reply:
Ah didn't note that. Thank you! Modified.